topics/galaxy-interface/tutorials/workflow-tips-and-tricks/tutorial.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: tutorial_hands_on
+
+title: Advanced tips and tricks for dealing with Galaxy Workflows
+tags:
+- workflows
+zenodo_link: ''
+questions:
+- Workflow?
+objectives:
+- Understand key aspects of workflows
+- Create clean, non-repetitive workflows
+requirements:
+  -
+    type: "internal"
+    topic_name: galaxy-interface
+    tutorials:
+      - workflow-editor
+time_estimation: 30m
+key_points:
+- Workflows
+contributors:
+- simonbray
+level: Intermediate
+subtopic: analyse
+---
+
+
+<!-- 
+
+Topics to cover:
+* Best practices panel
+* copying steps
+* PJAs
+* adding license/user metadata
+* how to resolve issues like converting a workflow to a collection type
+* updating steps
+ -->
+
+# Introduction
+{:.no_toc}
+
+
+Galaxy workflows combine multiple tools into a single analysis pipeline which can be executed with a single click, as if they were single tools. This tutorial discusses some more advanced features of the Galaxy workflow system, assuming you already have some basic knowledge of how to build workflows in Galaxy (e.g. by following the [Creating, Editing and Importing Galaxy Workflows]({{ site.baseurl }}/topics/galaxy-interface/tutorials/workflow-editor/tutorial.html)).
+
+We will start from the workflow created in that tutorial. You can either follow that tutorial to create the workflow, or import it as described below.
+
+## Importing a Galaxy workflow
+
+> ### {% icon hands_on %} Hands-on: Import a workflow
+>
+> - Click on **Workflow** in the top panel of the Galaxy page
+> - On the top right you will see 2 buttons: **Create** and **Import**.
+> - To import an existing workflow, click on **Import**.
+> - Copy and paste the following URL into the `Archived Workflow URL` box:
+> ```
+> https://raw.githubusercontent.com/galaxyproject/training-material/main/topics/galaxy-interface/tutorials/workflow-editor/workflows/main_workflow.ga
+> ```
+> 
+> - Click on **Import Workflow**. The new workflow will appear in your list of workflows when you click on the **Workflow** tab.
+{: .hands_on}
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Workflow best practices
+
+Galaxy recommends certain best practices are followed when defining a new workflow - for example, annotating a workflow with a description, a list of authors and a license. The best practices can be reviewed in the workflow editor via the 'Best Practices Review' panel.
```

Comment:
maybe we should pull it out into a separate tutorial and provide a 'non-best practice' workflow as a hands-on activity, so we can walk through each best practice and why it's important and useful.